### ISSUE TITLE
too quiet :D

### ISSUE THREAD

**Issue by Zibri** [quiet/quiet-js#32]
Hi, I am playing with the normal "audible" profile.
I have noticed that the produced audio volume is very low.
Expecially on phone.
How can I maximize the volume without "clipping"?
I should use the "gain" value or the "attenuation" value? Or both?

**Comment by Zibri** [comment]
Update: I noticed I can set "gain" up to 0.5 but if higher the script errors out.
Is that the maximum?

**Comment by brian-armstrong** [comment]
Hello,

Considering that it's an annoying modem noise and that making it louder doesn't make it receive better, is there a good reason for the gain to be higher?

**Comment by Zibri** [comment]
yes: when playing from a phone speaker to a receiving computer it would be better to have a higher volume from the phone so the computer mic can get it without needing to put the phone very near to the mic... Even worse if using higher frequencies since the frequency response of most speaker drops as frequency is higher. I was studying this for a side project I am thinking about.

**Comment by Zibri** [comment]
If the sound is generated from a computer there is less problem because raising the volume amplifies it enough.. but from a phone, even at maximum volume, the result leaves much to be desired.

**Comment by brian-armstrong** [comment]
It's counter-intuitive but increasing the volume will not actually help on this issue, since it's multipath/echo that causes this, and a louder source just means louder echos. You can't outrun your own shadow :)

**Comment by Zibri** [comment]
so, what settings do you suggest for phone to pc comunication?
